docs/partial_source/deep_dive/0_navigating_the_code.rst

```diff
@@ -129,6 +131,15 @@ look something like the following, (explained in much more detail in the followi
         """
         return ivy.current_backend(x).my_func(x, axes, dtype=dtype, device=device, out=out)
 
+We follow the `Array API Standard convention`_ about positional and keyword arguments.
+
+* Positional parameters must be positional-only parameters. Positional-only parameters have no externally-usable name. When a method accepting positional-only parameters is called, positional arguments are mapped to these parameters based solely on their order.
+* Optional parameters must be keyword-only arguments
+
+This convention makes it easier for us to modify functions in the future. Keyword-only parameters will mandate the use
+of argument names when calling functions, and this will allow us the flexibility to extend the function behaviour in
+future releases without breaking forward compatibility.
```

Comment:
... compatibility. Similar arguments can be kept together in the argument list, rather than us needing to add these at the very end to ensure positional argument behaviour remains the same.